### ISSUE TITLE
Compile error, missing class: AbstractScanner

### ISSUE THREAD

**Issue by SeanRMunoz** [zeroturnaround/maven-jrebel-plugin#9]
Our project was building fine on 4/22/16, now we are getting the following compile error:

```
[INFO] ------------------------------------------------------------------------
[ERROR] BUILD ERROR
[INFO] ------------------------------------------------------------------------
[INFO] Internal error in the plugin manager executing goal 'org.zeroturnaround:jrebel-maven-plugin:1.1.6-SNAPSHOT:generate': Unable to load the mojo 'org.zeroturnaround:jrebel-maven-plugin:1.1.6-SNAPSHOT:generate' in the plugin 'org.zeroturnaround:jrebel-maven-plugin'. A required class is missing: org/codehaus/plexus/util/AbstractScanner
org.codehaus.plexus.util.AbstractScanner
````
It appears to be a transitive dependency issue, where we're getting the wrong version of `plexus-utils`. 

```
[DEBUG] Plugin dependencies for:

org.zeroturnaround:jrebel-maven-plugin:1.1.6-SNAPSHOT

are:

org.sonatype.plexus:plexus-build-api:jar:0.0.7:runtime
org.apache.maven:maven-plugin-api:jar:2.0.6:runtime
org.apache.maven:maven-project:jar:2.0.6:runtime
commons-io:commons-io:jar:1.4:runtime
commons-lang:commons-lang:jar:2.4:runtime
org.codehaus.plexus:plexus-utils:jar:1.1:runtime
...
[DEBUG]   commons-io:commons-io:jar:1.4:runtime (selected for runtime)
[DEBUG]   commons-lang:commons-lang:jar:2.4:runtime (selected for runtime)
[DEBUG]   org.codehaus.plexus:plexus-utils:jar:1.5.8:runtime (removed - nearer found: 1.1)
[DEBUG]   org.codehaus.plexus:plexus-utils:jar:1.1:runtime (selected for runtime)
````

Our work-around has been to add the following explicit dependency to this project's pom, build and re-deploy to our local snapshot repo:

```
    <dependency>
      <groupId>org.codehaus.plexus</groupId>
      <artifactId>plexus-utils</artifactId>
      <version>1.5.8</version>
    </dependency>

**Comment by sander347** [comment]
**JIRA mirror: JR-5837**

Hi Sean!

Thanks for the report! Our engineers are already having a look into this.

**Comment by SeanRMunoz** [comment]
Great, thank you.

**Comment by daniel-anjos-zeroturnaround** [comment]
Hi Sean, sorry for the delay - we have added the transitive dependency - could you try the latest SNAPSHOT? Thanks

**Comment by SeanRMunoz** [comment]
Hi Daniel, 

Just attempted building with your latest SNAPSHOT (jrebel-maven-plugin-1.1.6-20160503.125737-6) and was stopped with the following, which is new for us:

````
[INFO] ------------------------------------------------------------------------
[ERROR] BUILD ERROR
[INFO] ------------------------------------------------------------------------
[INFO] Error resolving version for 'org.zeroturnaround:jrebel-maven-plugin': Plugin requires Maven version 3.0
[INFO] ------------------------------------------------------------------------
````
Unfortunately, our project is running Maven v2.2.1. Is this a new requirement of the jrebel plugin?

Thanks,
Sean

**Comment by daniel-anjos-zeroturnaround** [comment]
Thanks for your feedback - I believe this issue #9 was partially caused by the changes required to fix issue #3 (some new dependencies required upgrade to Maven +3.0). I believe the best way to keep using JRebel in your Maven 2 project is to keep using the 1.1.5 release. 

We are working to release version 1.1.6 soon with only minor features - fix for #3 and updating rebel.xml to be generated under resources folder and following the new version (2.1) of  XSD, but those changes required to drop maven 2 support.

Let me know if that suits you and your team. 
Best wishes,
Daniel

**Comment by SeanRMunoz** [comment]
Thank you for your help and clarification, Daniel. I have backed us down to version 1.1.5 and all seems well for the moment. Not sure why we were using 1.1.6-SNAPSHOT to begin with? Once we've upgraded to Maven 3.0+, we'll try going back to 1.1.6. 

Love using JRebel and can't imagine my work day without it. Keep up the great work!

Thanks,
Sean
